### DISCUSSION TITLE
Data inversion issues in EMagPy

### DISCUSSION THREAD

**Discussion by Paola1p23** [community/community#88517] [New to GitHub]
Good morning, I've encountered issues with data inversion. I have very clean starting values but with significantly high conductivity measurements. What are the limits of the model? I used **Q as a forward model**. The processed data doesn't seem to converge. Also, is there a more detailed explanation for the forward Q, as there is no explanation in the articles?

**Comment by queenofcorgis** [comment]
Hi @Paola1p23 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on data inversion. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
